packages/babel-runtime-corejs3/helpers/esm/applyDecs2305.js

```diff
@@ -0,0 +1,143 @@
+import _typeof from "./typeof.js";
+import _Array$isArray from "core-js-pure/features/array/is-array.js";
+import _Object$getOwnPropertyDescriptor from "core-js-pure/features/object/get-own-property-descriptor.js";
+import _bindInstanceProperty from "core-js-pure/features/instance/bind.js";
+import _pushInstanceProperty from "core-js-pure/features/instance/push.js";
```

Comment:
I suspect these can be avoided or replaced with more precise polyfills, the automatically imported ones don't seem to be minimal.